.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: Android Pull Request and Merge
+name: build
```

Comment:
is this one required? not against it just asking

Reply:
just to make it shorter